utilities/ubi8-google-api-python-client/README.md

```diff
@@ -8,6 +8,39 @@ Useful for environments where these libraries/tools are needed such as:
 * local development
 * CI tools that do not require specific CI tooling configuraiton, such as GiLab Jobs.
 
+## Helper Scripts
+
+This image also contains custom helper scripts for interacting with the Google API.
+
+### upload-file-to-google-drive.py
+This script makes an easy way for uploading files to Google Drive into existing or new folders.
+
+Generic Example
+```bash
+python3 /scripts/upload-file-to-google-drive.py \
+  --file=${LOCAL_FILE_PATH_TO_UPLOAD} \
+  --drive-id="${GOOGLE_DRIVE_ID}" \
+  --parent-drive-folder-id="${GOOGLE_DRIVE_PARENT_FOLDER_ID}" \
+  --sub-folder-name="${GOOGLE_DRIVE_SUB_FOLDER_NAME_UNDER_PARENT_DRIVE_FOLDER_ID_TO_UPLOAD_FILE_TO}" \
+  --destination="${GOOGLE_DRIVE_FILE_NAME}" \
+  --dest-mime-type='${DESTINATION_FILE_TYPE}' \
+  --source-mime-type='${SOURCE_MIME_TYPE}' \
+  --credentials="${google_api_credentials_json}"
+```
+
+Upload DOCX and convert to GDOC
+```bash
+python3 /scripts/upload-file-to-google-drive.py \
+  --file=foo.docx \
+  --drive-id="${GOOGLE_DRIVE_ID}" \
```

Comment:
can you add a section to the readme as to how I'd get these envs?

Reply:
done.